src/Resources/skeleton/security/formLogin/_LoginMethodBody.tpl.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
The `_L` (capital `L`) looks weird, but I can see you did that on purpose. What was the reason?